### PR TITLE
PXC-3110 Updated current 8.0 documentation.

### PR DESCRIPTION
**Pull request by patrickbirch**
Updated the following files:
source/.res/text/encrypt_binlog.removing.txt
source/add-node.rst
source/features/highavailability.rst
source/features/multimaster-replication.rst
source/features/pxc-strict-mode.rst
source/glossary.rst
source/howtos/proxysql-v2.rst
source/howtos/proxysql.rst
source/howtos/ubuntu_howto.rst
source/howtos/upgrade_guide.rst
source/howtos/virt_sandbox.rst
source/install/apt.rst
source/install/yum.rst
source/intro.rst
source/limitation.rst
source/management/data_at_rest_encryption.rst
source/manual/certification.rst
source/manual/gcache_record-set_cache_difference.rst
source/manual/performance_schema_instrumentation.rst
source/manual/threading_model.rst
source/manual/xtrabackup_sst.rst
source/overview.rst
source/security/encrypt-traffic.rst
source/wsrep-files-index.rst
source/wsrep-provider-index.rst
source/wsrep-status-index.rst
source/wsrep-system-index.rst

### CODE REVIEW

**Review by borys-belinsky — APPROVED** [review]

File: doc/source/management/data_at_rest_encryption.rst [hunk 1]
```diff
@@ -78,7 +80,8 @@ successfully loaded.
 
 .. note:: PXC recommends same configuration on all the nodes of the cluster,
    and that also means all the nodes of the cluster should have keyring
-   configured. Mismatch in keyring configuration will not allow JOINER node to
+   configured. A mismatch in keyring configuration will not allow a JOINER
+node to
```
Comment (kathy-williamson) [inline doc/source/management/data_at_rest_encryption.rst]: Correct format

File: doc/source/security/encrypt-traffic.rst [hunk 2]
```diff
@@ -87,7 +87,7 @@ enables automatic configuration of SSL encryption there-by encrypting
 This variable is not dynamic and so cannot be changed on runtime. To
 enable automatic configuration of SSL encryption, set
 ``pxc-encrypt-cluster-traffic=ON`` in the the ``[mysqld]`` section of the
-:file:`my.cnf` file, and restart the cluster (by default it is disabled
+:file:`my.cnf` file, and restart the cluster (by default, it is disabled
```
Comment (kathy-williamson) [inline doc/source/security/encrypt-traffic.rst:90]: Where is the end )

**Review by kathy-williamson — COMMENTED** [review]
I added 1 comment - I think there is a format issue

File: doc/source/management/data_at_rest_encryption.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/management/data_at_rest_encryption.rst]: Fixed.

**Pull request closed without merging**